libdnf/advisory/advisory.cpp

```diff
@@ -100,6 +100,42 @@ std::string Advisory::get_severity() const {
     return severity ? std::string(severity) : std::string();
 }
 
+unsigned long long Advisory::get_buildtime() const {
+    unsigned long long buildtime = get_pool(base).lookup_num(id.id, SOLVABLE_BUILDTIME);
+    return buildtime;
+}
+
+std::string Advisory::get_title() const {
+    // SOLVABLE_SUMMARY is misnamed, it actually stores the title
+    const char * title = get_pool(base).lookup_str(id.id, SOLVABLE_SUMMARY);
+    return title ? std::string(title) : std::string();
```

Comment:
Instead of repeating the condition in several places, you can use our inline function `c_to_str` (from utils/string.hpp).

Reply:
Done